Frends.SFTP.WriteFile/Frends.SFTP.WriteFile/Frends.SFTP.WriteFile.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System.ComponentModel;
+using Renci.SshNet;
+
+#pragma warning disable 1591
+
+namespace Frends.SFTP.WriteFile
+{
+    public class SFTP
+    {
+
+        /// <summary>
+        /// Writes a file with SFTP connection.
+        /// Documentation: https://github.com/FrendsPlatform/Frends.SFTP.WriteFile
+        /// </summary>
+        /// <param name="connection">Transfer connection parameters</param>
+        /// <param name="source">Source file location</param>
+        /// <param name="destination">Destination directory location</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>Result object </returns>
```

Comment:
You should document the result object with its properties. Like `Result object {string FirstProperty, int SecondProperty, etc.}`. This is important due to automatically generated documentation to Frends Task Portal.

Reply:
Fixed